internal/distroregistry/distroregistry.go

```diff
@@ -24,7 +24,7 @@ var supportedDistros = []supportedDistro{
 	{rhel8.New, rhel8.NewHostDistro},
 	{rhel84.New, rhel84.NewHostDistro},
 	{rhel84.NewCentos, rhel84.NewCentosHostDistro},
-	{rhel85.New, rhel85.NewHostDistro},
+	{rhel85.New, rhel84.NewRhel85HostDistro}, // Replace once RHEL8.5 package actually have any image types
```

Comment:
You should replace `rhel85.New` with `rhel84.NewRhel85`, because the distro returned by `rhel85.New()` does not have any image types defined. This change will allow us to not only build images on 8.5, but also 8.5 itself.